core-web/libs/block-editor/src/lib/extensions/image-tabview-form/components/dot-image-card-list/dot-image-card-list.component.html

```diff
@@ -0,0 +1,39 @@
+<p-scroller
+    *ngIf="contentlets?.length && !loading; else loading ? loadingBlock : emptyBlock"
+    [itemSize]="110"
```

Comment:
Not to fix, but scrollHeight, lazy and scrollHeight prospect to moved to a directive as default values like dotScrollerBlockEditor